terraform/cloud-platform-components/resources/opa/policy-ingress-conflicts.yaml

```diff
@@ -0,0 +1,24 @@
+# This policy prevents Ingress objects in different namespaces from sharing the same hostname.
+
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: policy-ingress-conflicts
+  namespace: opa
+  labels:
+    openpolicyagent.org/policy: rego
+data:
+  ingress-conflicts.rego: |
+    package cloud_platform.admission
+
+    import data.kubernetes.ingresses
+
+    deny[msg] {
+        input.request.kind.kind = "Ingress"
+        input.request.operation = "CREATE"
```

Comment:
I think we should address `PATCH` and `PUT` as well here? This is for when an `Ingress` changes or I use `kubectl apply` to create it. From my notes we tried `re_match("(CREATE|PUT|PATCH)", input.request.operation)` but this should be tested thoroughly to make sure we cover all cases.

Reply:
Agree, updated accordingly